google/resource_dataflow_job_test.go

```diff
@@ -45,6 +45,24 @@ func TestAccDataflowJobRegionCreate(t *testing.T) {
 	})
 }
 
+func TestAccDataflowJobCreateWithServiceAccount(t *testing.T) {
+	t.Parallel()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDataflowJobDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccDataflowJobWithServiceAccount,
+				Check: resource.ComposeTestCheckFunc(
+					testAccDataflowJobExists(
+						"google_dataflow_job.big_data"),
```

Comment:
From what I can tell, this is just checking that the job exists, not that the service account was set successfully. Is this something that could get updated?